secant/Features/SendFlow/SendFlowStore.swift

```diff
@@ -107,12 +109,16 @@ extension SendFlowReducer {
             sendReducer,
             transactionAddressInputReducer,
             transactionAmountInputReducer,
-            memoReducer
+            memoReducer,
+            addMemoCheckCircleReducer
```

Comment:
Similarly, I would call the reducer simply `addMemoReducer` and omit the component's name.